cmd/commands/cluster.go

```diff
@@ -219,10 +279,16 @@ func createAddClusterKustomization(ingressUrl, contextName, server, csdpToken, v
 		Resources: []string{
 			resourceUrl,
 		},
+		NameSuffix: nameSuffix,
 	}
 	k.FixKustomizationPostUnmarshalling()
 	util.Die(k.FixKustomizationPreMarshalling())
-	return k
+	return k, nameSuffix
+}
+
+func getClusterResourcesNameSuffix() string {
+	now := time.Now()
+	return fmt.Sprintf("%d", now.UnixMilli())
```

Comment:
i think it will be simpler to just return `fmt.Sprintf("-%d", now.UnixMilli())`
(have the "-" in the suffix)

Reply:
it will save all those tiny changes you made everywhere with the names of all resources